documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -190,7 +190,7 @@ Default: `0`
 [[SD_OVERCURRENT_CHECK]]
 ==== `SD_OVERCURRENT_CHECK`
 
-Raspberry Pi 4, Raspberry Pi 5 and Raspberry Pi 500 only.
+Raspberry Pi 4, Raspberry Pi 5, Raspberry Pi 500, and Raspberry Pi 500+ only.
```

Comment:
@timg236 Can you confirm that this doesn't apply to Pi 400?

Reply:
This should be a specific list btw